zcash_client_sqlite/src/lib.rs

```diff
@@ -587,6 +591,18 @@ impl<C: Borrow<rusqlite::Connection>, P: consensus::Parameters> WalletRead for W
             &address.encode(&self.params),
         )
     }
+
+    fn transaction_data_requests(&self) -> Result<Vec<TransactionDataRequest>, Self::Error> {
```

Comment:
This method doesn't seem to be covered by tests.

Reply:
I am working on an additional commit that adds more comprehensive testing.